Corona-Warn-App/src/main/AndroidManifest.xml

```diff
@@ -79,14 +79,20 @@
 
             <intent-filter android:autoVerify="true">
                 <action android:name="android.intent.action.VIEW" />
-
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
 
                 <data
                     android:host="e.coronawarn.app"
                     android:scheme="https" />
+
+                <data
+                    android:host="s.coronawarn.app"
+                    android:pathPattern=".*"
```

Comment:
I think the filter here is not needed?  it works without it

Reply:
it does not work without

Reply:
Ok, for me I did work , maybe you can add it also the other link if that is made work on your device ? who know what other devices have the same behaviour .

Reply:
Should I do it in another PR ? or will you do it here? 
```
 <data  android:host="e.coronawarn.app"
android:pathPattern=".*"
```
             android:scheme="https" />

Reply:
Did you try both patterns? With and without an extra /

Reply:
Sorry, maybe I was not clear, I mean you to add `android:pathPattern=".*"` to other intent filter of the event registration link, if that  path pattern works on your device. I am suggesting to use the same both intent filter , what do you think ?

Reply:
Let's do it in separate PR. I use a Xiaomi and without .* it just opens the browser.